test/jdk/java/awt/Scrollbar/AquaLFScrollbarTest/ScrollBarBorderTest.java

```diff
@@ -0,0 +1,196 @@
+import javax.swing.JScrollBar;
```

Comment:
Please add standard copyright header to the test. You can borrow it from any existing openjdk test made by Oracle - just make sure to change the year to the current.